app/src/main/java/com/infomaniak/drive/ui/MainViewModel.kt

```diff
@@ -632,6 +633,17 @@ class MainViewModel(
         }
     }
 
+    fun switchToNextUser(callback: () -> Unit) {
+        viewModelScope.launch(Dispatchers.IO) {
```

Comment:
Couldnt this feat on 1 line ? 
We almost always write this onelined to gain an indent level